c/include/cccl/c/types.h

```diff
@@ -11,14 +11,14 @@
 #pragma once
 
 #ifndef CCCL_C_EXPERIMENTAL
-#  warning "C exposure is experimental and subject to change. Define CCCL_C_EXPERIMENTAL to acknowledge this warning."
-#else // ^^^ !CCCL_C_EXPERIMENTAL ^^^ / vvv CCCL_C_EXPERIMENTAL vvv
+#  error "C exposure is experimental and subject to change. Define CCCL_C_EXPERIMENTAL to acknowledge this notice."
+#endif
```

Comment:
Nitpick: We try to add comments to all `#endif` because that is an easy rule and we have some very complex conditionals

```suggestion
#endif // !CCCL_C_EXPERIMENTAL
```